src/test/kotlin/com/ekino/oss/errors/RestExceptionHandlerTest.kt

```diff
@@ -31,10 +33,22 @@ class RestExceptionHandlerTest {
 
   @Test
   fun should_get_ok_response() {
-    mockMvc.perform(get("$ROOT_PATH/ok"))
+    mockMvc.perform(get("$ROOT_PATH/ok?id=${UUID.randomUUID()}"))
       .andExpect(status().isOk)
   }
 
+  @Test
+  fun should_get_missing_parameter_error() {
+    mockMvc.perform(get("$ROOT_PATH/ok"))
+      .andExpect(status().isBadRequest)
+      .andExpect(jsonPath("$.status", `is`(HttpStatus.BAD_REQUEST.value())))
+      .andExpect(jsonPath("$.code", `is`("error.missing_parameter")))
+      .andExpect(jsonPath("$.message", `is`(HttpStatus.BAD_REQUEST.reasonPhrase)))
+      .andExpect(jsonPath("$.description", not(isEmptyOrNullString())))
+      .andExpect(jsonPath("$.errors", empty<Any>()))
+      .andExpect(jsonPath("$.service", `is`("myApp : GET /test/ok")))
```

Comment:
We could add JCV to improve test readability :)

Reply:
@leomillon I agree. I've created https://github.com/ekino/spring-boot-starter-errors/issues/12 to track this.